### ISSUE TITLE
New age groups (40+)

### ISSUE THREAD

**Issue by dreamhatx** [bombardier-gif/covid-vaccine-booking#494]
The `filter_centers_by_age` function in `utils.py`, the centers are classified with 2 age limits (18 and 45). But, now there are sessions with age limits 40 (maybe others also).

So, a person with >40 age will not be able to book the vaccine in those sessions.

**Comment by karanssh** [comment]
Just adding here, some states have other random age groups too : source: https://twitter.com/BloreVaccine/status/1401059091485585408

Making this little more generic would help, I think

**Comment by dreamhatx** [comment]
Yes, in the PR #495, I have changed from checking based on predefined age categories to just checking if the age of beneficiary is greate than center minimum age.

**Comment by dreamhatx** [comment]
As @Nakul93 has pointed out ([here](https://github.com/bombardier-gif/covid-vaccine-booking/pull/495#discussion_r647398442)) there is an error for booking a session with min_age_booking 18 for an individual with age 45. I have not been able to test this personally.

So, I think this fix (#495) would not be valid or any fix will be hard to implement as @karanssh pointed that ([here](https://github.com/bombardier-gif/covid-vaccine-booking/issues/494#issuecomment-856498455)), there are numerous age classes and it might be impossible to filter based all the age classes and is very hard to know how the different age classes is affected by the limits and errors. (like can a person with age 42 for booking in an 18+ session or only for a 40+ session)

But I still don't get the logic where 18+ session is not bookable by a person with age above 45. Hoping for a fix soon as there are a lot of 40+ centers in my area.

**Comment by Nakul93** [comment]
I suggest we keep this open for now. We might need to handle it in the future.
As of now there are very few states and centers that are using these varying age filters.

**Comment by mrms123** [comment]
Also, with the changed vaccination policy (free for all above 18) - we might see these categories going away in the near future

Do we have a wiki made - if we do, maybe we could leave a small note there on how to edit the age category in the code

**Comment by darthfishious** [comment]
now that maharashtra has a 30+ age group as well, could we please incorporate this?

**Comment by jishnu-001** [comment]
can someone help me edit the age categories in my region they changed all centres to `18 and above` so there is no slot available in `18-44` and `45 and above` categories . i want to book for a person above 45 2nd dose but script shows no center available, but cowin site shows centres.

![Screenshot 2021-07-13 134158](https://user-images.githubusercontent.com/83004520/125416600-3285e309-b203-4d2b-ba80-58336e3bb6f3.jpg)

![Screenshot 2021-07-13 134246](https://user-images.githubusercontent.com/83004520/125416616-f4c80624-088f-466b-a73d-a82937114139.jpg)

i tried editing the `filter_centers_by_age` function and made it 

`def filter_centers_by_age(resp, min_age_booking):`
`     if min_age_booking >= 45:`
`        center_age_filter = 18`
`    else:`
`        center_age_filter = 18`

will this fix my issue or do i need to do something more i am a noob so if someone can fix it and send me the new file its much appreciated !

@Nakul93 @mrms123 @dreamhatx
